packages/eslint-plugin/tests/rules/strict-boolean-expressions.test.ts

```diff
@@ -281,9 +281,34 @@ ruleTester.run('strict-boolean-expressions', rule, {
       `,
       errors: [
         {
-          messageId: 'strictBooleanExpression',
+          messageId: 'conditionErrorAny',
           line: 3,
-          column: 20,
+          column: 28,
+        },
+      ],
+    },
+    {
+      code: `
+        let num = 1;
+        let str = "foo"
+        let val = null;
+        let bool = true && (val || num || str);
+      `,
+      errors: [
+        {
+          messageId: 'conditionErrorNullish',
+          line: 5,
+          column: 29,
+        },
+        {
+          messageId: 'conditionErrorNumber',
+          line: 5,
+          column: 36,
+        },
+        {
+          messageId: 'conditionErrorString',
+          line: 5,
+          column: 43,
```

Comment:
with these tests, you should use the `batchedSingleLineTests` function to separate them out as separate tests.
That way if one of the lines fails, you get jest test failure for just that line, instead of all 4 lines at once.
Makes it a bit easier to debug regressions

Reply:
I added it to some tests which expected one error per line, but not the one you highlighted, because it expects all 3 errors at the last line.